satellite/satellitedb/overlaycache.go

```diff
@@ -575,24 +573,18 @@ func (cache *overlaycache) CreateStats(ctx context.Context, nodeID storj.NodeID,
 		return nil, Error.Wrap(errs.Combine(err, tx.Rollback()))
 	}
 
-	if startingStats != nil {
-		auditSuccessRatio, err := checkRatioVars(ctx, startingStats.AuditSuccessCount, startingStats.AuditCount)
-		if err != nil {
-			return nil, errAuditSuccess.Wrap(errs.Combine(err, tx.Rollback()))
-		}
+	if startingStats == nil {
+		// TODO: add possible sanity checking for alpha and beta values
```

Comment:
Why not adding them rather leaving a `TODO`? the amount looks that should be minimal

Reply:
that's fair! let me put a wip label on this PR-- this TODO was more of a reminder for myself to address within this PR.